hackclub.com.yaml

```diff
@@ -423,6 +423,17 @@ asiet:
   - ttl: 1
     type: CNAME
     value: kkharidev.github.io.
+asmgame:
+  - ttl: 1
+    type: ALIAS
+    value: hackclub.github.io/asmgame.
+  - ttl: 1
+    type: A
+    values:
+      - 185.199.108.153
+      - 185.199.109.153
+      - 185.199.110.153
+      - 185.199.111.153
```

Comment:
What's the purpose of this A record?

Reply:
![image](https://github.com/hackclub/dns/assets/19589006/bb011f8e-4a89-40ab-8882-a24619b31b16)
rdns brings up github, so my best guess would be like a backup A if alias wasn't supported/didn't work?

Reply:
Yep, I'm just following Github's docs although it might not be necessary.